common/views/components/BaseTabs/BaseTabs.tsx

```diff
@@ -0,0 +1,146 @@
+import { useState, useRef, useContext, KeyboardEvent } from 'react';
+import { AppContext } from '@weco/common/views/components/AppContext/AppContext';
+import styled from 'styled-components';
+import { classNames } from '@weco/common/utils/classnames';
+
+const TabList = styled.div.attrs({
+  role: 'tablist',
+})``;
+
+type TabProps = {
+  isActive: boolean;
+  tabPanelId: string;
+};
+
+const Tab = styled.button.attrs((props: TabProps) => ({
+  className: classNames({
+    'plain-button no-padding': true,
```

Comment:
Did we say we're keeping styling out of this component?

Reply:
These buttons are an inherent part of the component, but by default they have button-y styles that we don't want. I guess the subtle difference is rather than _keeping styling out of this component_, we're _removing any styles from this component_.